components/notebook-controller/pkg/controller/notebook/notebook_controller.go

```diff
@@ -204,6 +214,12 @@ func (r *ReconcileNotebook) Reconcile(request reconcile.Request) (reconcile.Resu
 		if err != nil {
 			return reconcile.Result{}, err
 		}
+		log.Info("Creating virtual service", "namespace", service.Namespace, "name",
```

Comment:
shouldn't you do another check for virtualService by itself?
This if block is when you don't see the "service"?

Reply:
done.